src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,34 @@
+import { FormEvent, MouseEventHandler, ReactNode, useEffect, useRef, useState } from 'react';
```

Comment:
서영님 호옥시 여기 import문에 있는 것들 추후에 쓰일 예정일까용...?👀

Reply:
헉 아닙니다!! 감사합니다 나현님! 수정해두겠습니다~!!

```suggestion
import { ReactNode} from 'react';
```